### ISSUE TITLE
Swiftformat fails for "case x is [String: String]:"

### ISSUE THREAD

**Issue by RabugenTom** [nicklockwood/SwiftFormat#24]
With v0.9.1
````swift
switch y {
	case x is [Key: Value]: 
        fallthrough
    default: break
}
````
fails to be parsed. It seems to be caused by the colon after `Key`.

**Comment by nicklockwood** [comment]
Thanks for reporting. I've managed to reproduce the issue, and will put up a fix shortly.

**Comment by nicklockwood** [comment]
Fixed in 0.9.2.
